packages/db/src/migrations/20250915152638_create_granule_duplicates_table_and_index.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => {
+  await knex.schema.createTable('granule_duplicates', (table) => {
```

Comment:
:ant: I'd like to take a shot at calling it something other than 'granule_duplicate' - the data structure we're tracking here is a granule version grouping - duplicate carries an overloaded connotation that might be unfortunate in user-space, and there's not necessarily  to tie the data structure to the use directly. 

How about granule_group/s table?